test/models/pool_test.rb

```diff
@@ -36,19 +41,100 @@ def test_recently_expired_does_not_get_long_expired_subscriptions
       assert_equal expired_subscriptions, all_subscriptions - [unexpired, long_expired]
     end
 
-    def test_find_by_organization_and_id
-      Resources::Candlepin::Pool.expects(:find).returns(nil)
-      Pool.any_instance.expects(:organization).returns(nil)
-      assert_raises(ActiveRecord::RecordNotFound) do
-        Pool.find_by_organization_and_id!(get_organization, 3)
-      end
-    end
-
     def test_systems
       active_pool = FactoryGirl.build(:katello_pool, :active)
       systems = [katello_systems(:simple_server)]
       System.expects(:all_by_pool).with(active_pool.cp_id).returns(systems)
       assert_equal active_pool.systems, systems
     end
+
```

Comment:
What about testing some of the other model level functions to ensure their correctness?